frappe/public/js/frappe/views/communication.js

```diff
@@ -207,9 +207,8 @@ frappe.views.CommunicationComposer = Class.extend({
 				}
 
 				content_field.set_value(content.join(''));
-				if(subject === "") {
-					subject_field.set_value(reply.subject);
-				}
+
+				subject_field.set_value(reply.subject);
```

Comment:
- Why replace subject with Email Template's subject ?
- Wont this change the subject of the whole email conversation ?